eng/pipelines/runtime-official.yml

```diff
@@ -20,8 +20,6 @@ trigger:
     - SECURITY.md
     - THIRD-PARTY-NOTICES.TXT
 
-pr: none
```

Comment:
This should indeed be `pr: none` this should never run on PRs. This is the official build yml.

Reply:
it won't run unless somebody with write access manually triggers it, do we want to forbid it?  was /is it possible to trigger this pipeline using AzDo interface?

Reply:
Se want to forbid it since it would fail if you run it from the public domain. There is not even a public pipeline pointing to it, so I would doubt azure devops would say there is no pipeline with that name. Anyway I think it is not a big of a deal since we can't run it anyway.

Reply:
There's no pipeline using the definition in public, so you can't trigger this on a PR. It's not wrong per se, but it's safer to not let it run in case someone accidentally tried to. (Not sure it would even run correctly as it will try to access a bunch of secrets and tasks only available internally)